server/repo-mgr.c

```diff
@@ -820,55 +824,39 @@ int
 set_repo_commit_to_db (const char *repo_id, const char *repo_name, gint64 update_time,
                        int version, gboolean is_encrypted, const char *last_modifier)
 {
```

Comment:
Why is this function changed?

Reply:
Because the sql 'REPLACE' will try to delete previous record then do 'INSERT' operation, in this situation, the 'status' column will be set to NULL.